src/util/config/config.cpp

```diff
@@ -111,6 +111,14 @@ namespace dxvk {
   void Config::merge(const Config& other) {
     for (auto& pair : other.m_options)
       m_options.insert(pair);
+
+    if (!m_options.empty()) {
+      // Inform the user about effective configuration settings
+      Logger::info("Effective config settings:");
+      for (auto& pair : m_options) {
```

Comment:
Just a bit of nitpicking, but please don't use `{ }` around single-line blocks for consistency.

Reply:
Done.